src/GafferSceneUIModule/SceneGadgetBinding.cpp

```diff
@@ -132,6 +134,17 @@ IECore::InternedStringVectorDataPtr objectAt( SceneGadget &g, IECore::LineSegmen
 	return nullptr;
 }
 
+tuple objectAndIntersectionAt( SceneGadget &g, IECore::LineSegment3f &l )
+{
+	IECore::InternedStringVectorDataPtr result = new IECore::InternedStringVectorData;
+	Imath::V3f hitPos( 0.0f );
+	if( g.objectAt( l, result->writable(), hitPos ) )
+	{
+		return boost::python::make_tuple( true, result, hitPos );
```

Comment:
Why three? The original binding is returning a single value (`None` when there's no intersection), so shouldn't we return a tuple with that same value, plus the hitPos?

Reply:
Do you mean `(None, hitPoint)` or `None` for the no hit scenario? (I'm less of a fan of varying return types, and in this case it means you can't easily use destructuring assignments).

Reply:
> Do you mean (None, hitPoint) or None for the no hit scenario?

I mean `( None, hitPoint )`. I'm just talking about omitting the additional `true/false` tuple item.

Reply:
A fine point indeed, corrected in https://github.com/GafferHQ/gaffer/pull/3403/commits/033df9c21859f7832051f1876b329d9e7d9de66f